Backbase/variants
# feature: Expose configuration variables key to Variants.swift

### What does this PR do
Based on `variants.yml` generate in `Variants.swift` if needed enum `ConfigurationValueKey` with configuration variable's names as keys. Now, to get configuration value you can use next code: 
 ```Variants.configurationValue(key: .A_PROPERTY)```

While old way is kept as well
```Variants.configuration["A_PROPERTY"]```

### How can it be tested
Add into yml custom not environment properties with destination project and try command `variants setup`. In generated `Variants.swift` you should found all properties as enum cases

### Task
resolves #125  

### Checklist:
